simtools/model/camera.py

```diff
@@ -177,21 +176,13 @@ def _rotatePixels(self, pixels):
             The pixels dictionary with rotated pixels.
             The pixels orientation for plotting is added to the dictionary in pixels['orientation'].
             The orientation is determined by the pixel shape (see readPixelList for details).
-
-        Notes
-        -----
-        The additional rotation angle to get to the camera view of an observer facing the camera
-        is saved in the const dictionary CAMERA_ROTATE_ANGLE.
-        In the case of dual mirror telescopes, the axis is flipped in order to keep the same
-        axis definition as for single mirror telescopes.
-        One can check if the telescope is a two mirror one with isTwoMirrorTelescope.
         """
 
-        if isTwoMirrorTelescope(self._telescopeModelName):
-            pixels["y"] = [(-1) * yVal for yVal in pixels["y"]]
-
         rotateAngle = pixels["rotateAngle"]  # So not to change the original angle
```

Comment:
What does the comment 'so not to change...' mean? Is it that we don't want to change `pixels['rotationAngle']`? If yes, then the comment should say this.

Reply:
In the next line I add 90 degrees to the rotation angle. I don't want to do `pixels["rotateAngle"] += np.deg2rad(90)` so not to change the value in `pixels["rotateAngle"]` (might be used later). That comment is so trivial that it might be confusing, so maybe I should remove it. I will think about it.